style.css

```diff
@@ -22,31 +24,86 @@ body {
   flex-grow: 0;
 }
 
-header {
-  height: 95%;
-  width: 95%;
-  border-radius: 0.25rem;
+.navbar {
   display: flex;
   justify-content: space-between;
   align-items: center;
-  background-color: white;
+  background-color: #396df2;
+  color: white;
 }
 
 .Logo {
-  width: fit-content;
-  font-style: normal;
-  color: #6070ff;
-  font-size: 18px;
-  line-height: 20px;
-  text-align: center;
-  font-weight: 700;
-  margin: 10px 12px;
+  font-size: 1.5rem;
+  margin: 0.5rem;
+}
+
+.navbar-links ul {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.navbar-links li {
+  list-style: none;
+}
+
+.navbar-links li a {
   text-decoration: none;
+  color: white;
+  padding: 1rem;
+  display: block;
+}
+
+.navbar-links li a:hover {
+  background-color: #11256f12;
+}
+
+.toogle-button {
+  position: absolute;
+  top: 0.75rem;
+  right: 1rem;
+  display: none;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 30px;
+  height: 21px;
 }
 
-.Navbar {
-  width: 18px;
-  height: 12px;
+.toogle-button .bar {
+  height: 3px;
+  width: 100%;
+  background-color: white;
+  border-radius: 10px;
+}
+
+.overlay {
+  display: none;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 3;
+  background-color: #0920ea;
+  backdrop-filter: blur(3);
+  opacity: 0.75;
+}
```

Comment:
- [ ] Kindly consider changing your background-color to match that of the `Figma` design. The background-color for the `Figma` design is `#6070FF`. This is to make sure your design looks close to the `Figma` design as much as possible.
- [ ] Also to create a good `user-experience` it would be better if you set your `backdrop-filter` style properly as it's currently not taking effect. This is because you didn't specify a `unit` for the `blur`. It should look like this `backdrop-filter: blur(3px)`. 

<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="372" alt="Screenshot 2023-05-23 at 20 29 13" src="https://github.com/amiralda/amiral-portfolio/assets/107192076/d8af3199-f904-4742-bd06-042cdefaca3f">
</td>
<td>
<img width="372" alt="Screenshot 2023-05-23 at 20 28 31" src="https://github.com/amiralda/amiral-portfolio/assets/107192076/28f19f9a-8cce-4a80-98c7-2e3fc22df32c">
</td>
</tr>
</table>

**📝 Note:** Please make sure that the nav text and the background color contrast are good enough that users can easily see the nav text without the background interfering.